include/nbl/video/ILogicalDevice.h

```diff
@@ -105,16 +105,19 @@ class NBL_API2 ILogicalDevice : public core::IReferenceCounted, public IDeviceMe
                 // https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkImageMemoryBarrier2-srcStageMask-03854
                 constexpr auto HostBit = asset::PIPELINE_STAGE_FLAGS::HOST_BIT;
                 if (barrier.dep.srcStageMask.hasFlags(HostBit)||barrier.dep.dstStageMask.hasFlags(HostBit))
+                    m_logger.log("Source and destination queue family index should be equal, HOST_BIT is set [%s - %s:%p]", system::ILogger::ELL_ERROR, __FUNCTION__, __FILE__, __LINE__);
```

Comment:
better log message, source and dest queue indices are a vulkan thing, we only have an other queue index and a ownership op type

the real reason is that you can't do an ownership transfer that syncs against a host stage

Reply:
changed to `"Invalid barrier, ownership transfer with host is not allowed"`